webpack.config.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
       {
         test: /\.php$/,
         loader: 'transform?phpify'
+      },
+      {
```

Comment:
I can't wait until we can turn PHPify into a proper Webpack loader that handles source maps the Webpack way, so we can get rid of this, but for now 👏